torch/_export/passes/replace_sym_size_ops_pass.py

```diff
@@ -16,10 +16,11 @@ class _ReplaceSymSizeOpPass(PassBase):
     and torch.ops.aten.sym_stride with torch.ops.aten.sym_stride.int
     """
 
-    def call(self, graph_module):
+    def call(self, graph_module) -> PassResult:
         for module in graph_module.modules():
             if not isinstance(module, torch.fx.GraphModule):
                 continue
             for node in module.graph.nodes:
                 if node.target in replacements:
                     node.target = replacements[node.target]
+        return PassResult(graph_module, True)
```

Comment:
why always `True`?

Reply:
We shouldn't do that. Thanks for pointing out!